src/codewhisperer/util/telemetryHelper.ts

```diff
@@ -35,12 +35,23 @@ export class TelemetryHelper {
 
     public startUrl: string | undefined
 
+    // variables for client component latency
+    private invokeSuggestionStartTime = 0
+    private fetchCredentialStartTime = 0
+    private sdkApiCallStartTime = 0
```

Comment:
why there's still sdk latency measured? We said we are not going to measure this in the doc.

Reply:
Hi Andrew, sdk api call start time is only used to compute the other relevant latencies, it is not used to compute SDK latency.